docs/opcodes/55/berlin.mdx

```diff
@@ -1,5 +1,10 @@
 ## Gas
 
+### Definitions:
+- **value**: The value of the storage slot after executing the opcode SSTORE.
```

Comment:
I think that's not clear. How about "the one from the stack input"? Also to stay more clear with the "estimate" section.

Reply:
Changed it to a code section